### PR TITLE
Use only Daml Script in Explicit Disclosure docs example

### PR DESCRIPTION
**Pull request by tudor-da**

### CODE REVIEW

File: docs/3.0.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 1]
```diff
@@ -248,271 +248,20 @@ In the example above, **Buyer** does not have visibility over the ``stockCid``,
...
-This example uses the following Canton participant node topology:
+       disclosedStock <- fromSome <$> queryDisclosure stockExchange stockCid
+       disclosedOffer <- fromSome <$> queryDisclosure seller offerCid
+       disclosedPriceQuotation <- fromSome <$> queryDisclosure stockExchange priceQuotationCid
 
-- ``stockExchangeParticipant`` hosts party with display name ``StockExchange`` and exposes Ledger API on port ``5011``
-- ``bankParticipant`` hosts party with display name ``Bank`` and exposes Ledger API on port ``5021``
-- ``buyerParticipant`` hosts party with display name ``Buyer`` and exposes Ledger API on port ``5031``
-- ``sellerParticipant`` hosts party with display name ``Seller`` and exposes Ledger API on port ``5041``
+Then, the **Buyer** attaches the disclosed contract payloads to the command submission in which he accepts the offer.
```
Comment (mziolekda) [inline docs/3.0.0/docs/app-dev/explicit-contract-disclosure.rst:257]:
```suggestion
Then, the **Buyer** attaches the disclosed contract payloads to the command submission that accepts the offer.
```

File: docs/2.7.6/docs/app-dev/explicit-contract-disclosure.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 .. _explicit-contract-disclosure:
 
-Explicit Contract Disclosure (Alpha)
+Explicit Contract Disclosure (Beta)
```
Comment (marciw-da) [inline docs/2.7.6/docs/app-dev/explicit-contract-disclosure.rst]: @hrischuk-da Can we do this? Or does this technically trigger 2.7.7? Tudor asked me about this but I don't know.

(@tudor-da I haven't done a full review yet; just asking this Q for now)

Comment (marciw-da) [reply docs/2.7.6/docs/app-dev/explicit-contract-disclosure.rst]: Oh, I guess I was looking at an outdated file; you can disregard my comment

File: docs/3.0.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 3]
```diff
@@ -248,271 +248,20 @@ In the example above, **Buyer** does not have visibility over the ``stockCid``,
...
-  }
-  EOF
-  )
-
-  iouTransactions=$(grpcurl -plaintext -d $iouQuery localhost:5031 com.daml.ledger.api.v1.TransactionService/GetTransactions)
-
-**Buyer** receives these contracts from the stakeholders and adapts them to disclosed contracts (as described in :ref:`the previous section <submitter-disclosed-contract>`)
-in a command submission that executes ``Offer_Accept`` on the ``offerCid``. The resulting gRPC command submission, which succeeds, is
-shown below.
+       -- -- Submission using disclosed contracts
```
[the remark this answers is not in the document]
Comment (tudor-da) [reply docs/3.0.0/docs/app-dev/explicit-contract-disclosure.rst]: Good point about the comment, let's delete it.

**Review by mziolekda — APPROVED** [review]

**Review by marciw-da — APPROVED** [review]
Thanks for this update!

In case anyone sees this and wonders why we're updating 3.0.0: this is the explicit disclosure doc that needs to move from 3.0.0 to 2.9.0 but currently exists only in 3.0.0. Now that we have these updates, I'll move this doc to 2.9.0 and we'll make any other changes there.

**Pull request merged**